### DISCUSSION TITLE
Is there any timeline on releasing version 0.7.0 of sheldon?

### DISCUSSION THREAD

**Discussion by denysdovhan** [rossmacarthur/sheldon#148] [Q&A] [answered]
@rossmacarthur I'm looking forward on `profile` feature. I see those changes in the `trunk` branch, but there is still no release published. Any timeline on when it's going to be formally released?

**Comment by rossmacarthur** [comment, marked as answer]
I would love to release this soon. After using the profile feature locally I encountered the following issue which I think might need to resolved before releasing this feature.

https://github.com/rossmacarthur/sheldon/issues/147

**Reply by denysdovhan** [reply]
Thanks! I've subscribed to the issue.

Personally, applying plugins from a single lockfile depending on a profile seems more reasonable to me.
